dist/ruff.toml

```diff
@@ -0,0 +1,30 @@
+required-version = "==0.4.5"
+
+extend-exclude = [
+    "../test/3rdparty",
+]
+
+[lint]
+ignore = [
+    "E401", # multiple-imports-on-one-lines
+    "E402", # module-import-not-at-top-of-file
+    "E701", # multiple-statements-on-one-line-colon
+    "E702", # multiple-statements-on-one-line-semicolon
```

Comment:
Thanks for adding the error names! Much easier to work with than googling error numbers